distro/configuration/conceptsources/conceptsources.csv

```diff
@@ -52,3 +52,4 @@ b553904d-23c0-5394-b115-bc132505c053,,HL7-ActSubstanceAdminSubstitutionCode,HL7-
 23c66054-1469-5486-ba22-52b0b89d876d,,ICD-10-WHO-NP2,ICD-10-WHO-NP2,,
 df272316-641f-5690-99fa-937df27a81f5,,HL7-DiagnosticReportStatus,HL7-DiagnosticReportStatus,,
 4d85531c-f351-5f85-aae2-2856e5f51e11,,CCAM,CCAM,,
+f6417e5c-4e0c-5b54-96f9-65ba241f0949,,OS4H.MASTER,Imported from https://api.openconceptlab.org/orgs/OHRITechGroup/sources/TB_SC/mappings/1/,,
```

Comment:
Hey @smallgod are u still making more additions? we could add them at once before merging this in

Reply:
naaa, that's it! Please merge